docs/api_specification.md

```diff
@@ -35,39 +113,13 @@ Content-Type: application/json
 ### Response example
 
 ```
-201 OK
+200 OK
```

Comment:
Is it not `201 OK` https://github.com/alphagov/pay-adminusers/blob/4cfd993b5652300f45afd0ef40b33c574a091c3b/src/main/java/uk/gov/pay/adminusers/resources/UserResource.java#L126 ?

Reply:
Good spot, I must have been looking at the wrong one, I'll fix